app/views/layouts/_layout.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="container">
+    <article class="main-content">
+    	<%= yield %>
+    </article>
+    <aside class="secondary-content">
+      <section class="container">
+		  <%= render partial: "shared/sidebar" if user_signed_in? %>
```

Comment:
this line probably doesn't need the `if user_signed_in?` presumably because the user needs to be signed in to access this layout.